Lib/test/test_webbrowser.py

```diff
@@ -265,6 +267,23 @@ def test_register_default(self):
     def test_register_preferred(self):
         self._check_registration(preferred=True)
 
+    def test_environment_variable_BROWSER_is_respected(self):
```

Comment:
This test does accurately catch the regression as far as I can tell but it seems a little obscure. Thoughts?